Doc/library/multiprocessing.rst

```diff
@@ -2305,14 +2305,11 @@ multiple connections at the same time.
    generally be omitted since it can usually be inferred from the format of
    *address*. (See :ref:`multiprocessing-address-formats`)
 
-   If *authenticate* is ``True`` or *authkey* is a byte string then digest
-   authentication is used.  The key used for authentication will be either
-   *authkey* or ``current_process().authkey`` if *authkey* is ``None``.
-   If authentication fails then
-   :exc:`~multiprocessing.AuthenticationError` is raised.  See
-   :ref:`multiprocessing-auth-keys`.
+   If *authkey* is a byte string, digest authentication is used.
```

Comment:
I'd like to suggest an improvement: `If *authkey* is given, it should be a byte string and will be used as the secret key for an authentication challenge.`

Also how about improving the `multiprocessing-auth-keys` section to mention the challenge is HMAC-based?

Reply:
I reworded both sections (for Client and Listener) to use that sentence.

Also, `multiprocessing-auth-keys` already mentions HMAC; I added it to the Client and Listener docs.